packages/libp2p/src/dcutr/dcutr.ts

```diff
@@ -117,21 +113,73 @@ export class DefaultDCUtRService implements Startable {
   }
 
   /**
-   * Perform the inbound connection upgrade as B
+   * This is the callback function for the onConnect handler of {@link multicodec} connections.
+   * We make sure that the connection is an inbound relayed connection and then initiate the upgrade.
    *
-   * @see https://github.com/libp2p/specs/blob/master/relay/DCUtR.md#the-protocol
+   * The protocol starts with the completion of a relay connection from A to B.
+   *
+   * @param {PeerId} peerId
+   * @param {Connection} connection
+   * @returns
```

Comment:
These auto-generated JSDoc comments add nothing that TypeDoc can't get from TSC, please remove.

Reply:
editor added. my bad